V1.0.0 RC (#182)
OWASP/maswe: weaknesses/MASVS-PRIVACY/MASWE-0078.md

Author: Carlos: DONE

Author: @TheDauntless this is the new one that we decided to keep separate from MASWE-0071	Inadequate or Ambiguous User Consent Mechanisms, could you please review and improve if needed?

---
title: Inadequate Awareness for Privacy Relevant Actions
id: MASWE-0078
alias: inadequate-awareness-privacy-actions

> requirement: "Before completing a privacy relevant action, the app clearly communicates the affected data, destination, audience, and material privacy consequences, and allows the user to review and control the resulting disclosure."

Reviewer: ```suggestion
requirement: "The app informs the user about the privacy implications before performing an action."
```
